### DISCUSSION TITLE
Allow server listings

### DISCUSSION THREAD

**Discussion by EpicPlayerA10** [Minestom/Minestom#617] [Help] [answered]
Is there a way to get `Allow server listings` property from player? I can't find it.

**Comment by emortaldev** [comment, marked as answer]
Minestom does not appear to have this. You would normally do it with `player.getSettings()` e.g. `player.getSettings().getLocale()`

**Reply by MrlnHi** [reply]
If smortal did it right #620 should fix this

**Reply by emortaldev** [reply]
Fixed in f1ca048
